.github/workflows/ci.yml

```diff
@@ -21,3 +21,16 @@ jobs:
 
     - name: Run tests
       run: yarn run test
+
+  coverage:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: 15.x
+        
+    - run: yarn install
+    - run: yarn test
+    - run: bash <(curl -s https://codecov.io/bash)
```

Comment:
Codecov has an npm tool that can be run with `npx codecov` so you don't have to bash this directly

Reply:
I'll look into it. Thank you.

Reply:
Did you look into it?